### PR TITLE
Adds support for storage volume templates

### PR DESCRIPTION
**Pull request by andreadean5**

### CODE REVIEW

File: examples/storage_volume_templates.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+# Find or add storage pool to use in template
+print("Find or add storage pool to use in template")
+storage_pools = oneview_client.storage_pools.get_all()
+storage_pool_added = False
+storage_system_added = False
+if storage_pools:
+    storage_pool = storage_pools[0]
+    print("   Found storage pool '{}' at uri: '{}".format(
+        storage_pool['name'], storage_pool['uri']))
```
Comment (balestrinc) [inline examples/storage_volume_templates.py]: I think is more readable format the strings like this:   
`"   Found storage pool '{name}' at uri: '{uri}".format(**storage_pool)`

File: examples/storage_volume_templates.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+else:
+    # Find or add storage system
+    storage_pool_added = True
+    print("   Find or add storage system")
+    s_systems = oneview_client.storage_systems.get_all()
+    if s_systems:
+        s_system = s_systems[0]
+        storage_system_added = False
+        print("      Found storage system '{}' at uri: {}".format(s_system['name'], s_system['uri']))
```
Comment (balestrinc) [inline examples/storage_volume_templates.py]: the same

File: examples/storage_volume_templates.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        options_storage = {
+            "ip_hostname": config['storage_system_hostname'],
+            "username": config['storage_system_username'],
+            "password": config['storage_system_password']
+        }
+        s_system = oneview_client.storage_systems.add(options_storage)
+        s_system['managedDomain'] = s_system['unmanagedDomains'][0]
+        s_system = oneview_client.storage_systems.update(s_system)
+        storage_system_added = True
+        print("      Added storage system '{}' at uri: {}".format(s_system['name'], s_system['uri']))
```
Comment (balestrinc) [inline examples/storage_volume_templates.py]: the same

File: examples/storage_volume_templates.py (hunk 1 above)
Comment (andreadean5) [inline examples/storage_volume_templates.py]: got it!

**Pull request merged**